### ISSUE TITLE
User Friendly Build

### ISSUE THREAD

**Issue by EMLVIRUS** [aria2/aria2#838]
Right now, Aria2 is still very unfriendly to many users though it is indeed an early project.
But at least a complete GUI should be easy to be prepared just like IDM, FDM, etc.

**Comment by gbcox** [comment]
There are front-end apps which can help with that...
1.  Chrome Aria2c integration:
https://chrome.google.com/webstore/detail/aria2c-integration/cnkefpcjiolhnmhfpjbjpidgncnajlmf?utm_source=chrome-app-launcher-info-dialog
2.  webui-aria2
https://github.com/ziahamza/webui-aria2
3.  uGet Download Manager
http://ugetdm.com/

Those are just a few... I'm sure there are others.

I wouldn't choose the word "unfriendly".  Is it powerful?  Yes... Does it have alot of options you can use?  Yes... IMHO that is the attractiveness of the application - but you don't have to tweak it... you can just use the defautts.  If you begin to "dumb it down" you might as well use one of the other many FTP apps which are available.

**Comment by HulkTan** [comment]
Well actually I asked a friend of mine to write a beginner's setup.exe to help you setup aria2 for the first time. You can get it here:
https://mega.nz/#!r0FmjSKR!j8ASyCL6HilyhJNf_rnMzunN8VjX3XSE4TsJdni9U7U
